cmd/sqlflowserver/main_test.go

```diff
@@ -147,6 +151,33 @@ func prepareTestData(dbStr string) error {
 	return fmt.Errorf("unrecognized SQLFLOW_TEST_DB %s", os.Getenv("SQLFLOW_TEST_DB"))
 }
 
+func generateCA(out string) (string, string, error) {
+	caKey := path.Join(out, "ca.key")
+	caCsr := path.Join(out, "ca.csr")
+	caCrt := path.Join(out, "ca.crt")
+	if err := exec.Command("openssl", "genrsa", "-out", caKey, "2048").Run(); err != nil {
+		return "", "", err
+	}
+	if err := exec.Command("openssl", "req", "-nodes", "-new", "-key", caKey, "-subj", "/CN=localhost", "-out", caCsr).Run(); err != nil {
+		return "", "", err
+	}
+	if err := exec.Command("openssl", "x509", "-req", "-sha256", "-days", "365", "-in", caCsr, "-signkey", caKey, "-out", caCrt).Run(); err != nil {
+		return "", "", err
+	}
+	os.Setenv("SQLFLOW_CA_CRT", caCrt)
+	os.Setenv("SQLFLOW_CA_KEY", caKey)
+	return caCrt, caKey, nil
+}
+
+func CreateRPCConn() (*grpc.ClientConn, error) {
```

Comment:
If CreateRPCConn is going to be used out of this unit test, it should be moved to somewhere else. Otherwise, it should not be an exported symbol and should be renamed to createRPCConn or connect.

Reply:
Thanks, done. `createRPCConn ` is only used in `main_test.go`